External/FEXCore/Source/Interface/Core/X86Tables/VEXTables.cpp

```diff
@@ -394,7 +394,7 @@ void InitializeVEXTables() {
     {OPD(2, 0b11, 0xF3), 1, X86InstInfo{"", TYPE_VEX_GROUP_17, FLAGS_NONE, 0, nullptr}}, // VEX Group 17
 
     {OPD(2, 0b00, 0xF5), 1, X86InstInfo{"BZHI", TYPE_INST, FLAGS_MODRM | FLAGS_VEX_2ND_SRC, 0, nullptr}},
-    {OPD(2, 0b01, 0xF5), 1, X86InstInfo{"PEXT", TYPE_UNDEC, FLAGS_NONE, 0, nullptr}},
+    {OPD(2, 0b10, 0xF5), 1, X86InstInfo{"PEXT", TYPE_INST, FLAGS_MODRM | FLAGS_VEX_1ST_SRC, 0, nullptr}},
```

Comment:
Can we put a log on the line before that the AMD reference manual is incorrect with its instruction tables?
AMD instruction tables claim the instruction maps to 0b01, implementation pages correctly claim 0b10

Reply:
Done